provision/minikube/keycloak/templates/keycloak.yaml

```diff
@@ -71,27 +65,6 @@ spec:
       value: 'false'
     - name: spi-public-key-cache-infinispan-enabled
       value: 'false'
-{{ end }}
-{{ if eq .Values.storage "jpa+hotrod" }}
-    # in the mixed setup we set storage to jpa and override user sessions, auth sessions, single-use objects, and login failures to use hotrod
-    - name: storage
-      value: jpa
-    - name: storage-area-auth-session
-      value: hotrod
-    - name: storage-area-user-session
-      value: hotrod
-    - name: storage-area-single-use-object
-      value: hotrod
-    - name: storage-area-login-failure
-      value: hotrod
-{{ else if ne .Values.storage "" }}
```

Comment:
Would this configuration be not valid anymore ?

Reply:
All storage configurations are removed, so it is no longer valid.